### PR TITLE
Add Local Windows Group Members

### PR DESCRIPTION
**Pull request by flynngw**
Now reports on the members of the local Windows groups

## Description

Adds Local Windows Group Members into the output of the AsbuiltReports for Windows
So will now output Group Name, Description, Members for Local Groups (rather than just Group Name, Description)

## Related Issue

Minor new feature

## Motivation and Context

Added important detail for an Asbuilt document - local group memberships need to be documented.

## How Has This Been Tested?

Have tested this code in several of my environments. Only a minor change and works without issue.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the [**CONTRIBUTING**](/CONTRIBUTING.md) document.

### CODE REVIEW

File: Src/Public/Invoke-AsBuiltReport.Microsoft.Windows.ps1 [hunk 1]
```diff
@@ -5,7 +5,7 @@ function Invoke-AsBuiltReport.Microsoft.Windows {
     .DESCRIPTION
         Documents the configuration of Microsoft Windows Server in Word/HTML/Text formats using PScribo.
     .NOTES
-        Version:        0.4.0
+        Version:        0.4.1
```
Comment (rebelinux) [inline Src/Public/Invoke-AsBuiltReport.Microsoft.Windows.ps1]:
```suggestion
        Version:        0.5.2
```

File: Src/Private/Get-AbrWinLocalGroup.ps1 [hunk 2]
```diff
@@ -5,7 +5,7 @@ function Get-AbrWinLocalGroup {
     .DESCRIPTION
         Documents the configuration of Microsoft Windows Server in Word/HTML/Text formats using PScribo.
     .NOTES
-        Version:        0.2.0
+        Version:        0.2.1
```
Comment (rebelinux) [inline Src/Private/Get-AbrWinLocalGroup.ps1]:
```suggestion
        Version:        0.5.2
```

File: Src/Public/Invoke-AsBuiltReport.Microsoft.Windows.ps1 [hunk 3]
```diff
@@ -110,7 +110,7 @@ function Invoke-AsBuiltReport.Microsoft.Windows {
             if ($InfoLevel.Account -ge 1) {
                 try {
                     $LocalUsers = Invoke-Command -Session $TempPssSession { Get-LocalUser | Where-Object {$_.PrincipalSource -ne "ActiveDirectory"} }
-                    $LocalGroups = Invoke-Command -Session $TempPssSession { Get-LocalGroup | Where-Object {$_.PrincipalSource -ne "ActiveDirectory" }}
+					$LocalGroups = Invoke-Command -Session $TempPssSession { Get-LocalGroup | Where-Object {$_.PrincipalSource -ne "ActiveDirectory" } | ForEach-Object { [PSCustomObject]@{ GroupName = $_.Name;  Description = $_.Description; Members = (Get-LocalGroupMember -Group $_.Name).Name } }}
```
Comment (rebelinux) [inline Src/Public/Invoke-AsBuiltReport.Microsoft.Windows.ps1]: You need to fix the text alignment

File: Src/Private/Get-AbrWinLocalGroup.ps1 [hunk 4]
```diff
@@ -33,8 +33,9 @@ function Get-AbrWinLocalGroup {
                         ForEach ($LocalGroup in $LocalGroups) {
                             try {
                                 $TempLocalGroupsReport = [PSCustomObject]@{
-                                    'Group Name' = $LocalGroup.Name
+                                    'Group Name' = $LocalGroup.GroupName
                                     'Description' = $LocalGroup.Description
+									'Members' = $LocalGroup.Members
```
Comment (rebelinux) [inline Src/Private/Get-AbrWinLocalGroup.ps1]: You need to fix the text alignment

**Review by rebelinux — CHANGES_REQUESTED** [review]

File: Src/Public/Invoke-AsBuiltReport.Microsoft.Windows.ps1 (hunk 1 above)
Comment (flynngw) [reply Src/Public/Invoke-AsBuiltReport.Microsoft.Windows.ps1]: Resolved

File: Src/Private/Get-AbrWinLocalGroup.ps1 (hunk 2 above)
Comment (flynngw) [reply Src/Private/Get-AbrWinLocalGroup.ps1]: Resolved

File: Src/Private/Get-AbrWinLocalGroup.ps1 (hunk 4 above)
Comment (flynngw) [reply Src/Private/Get-AbrWinLocalGroup.ps1]: Resolved

File: Src/Public/Invoke-AsBuiltReport.Microsoft.Windows.ps1 (hunk 3 above)
Comment (flynngw) [reply Src/Public/Invoke-AsBuiltReport.Microsoft.Windows.ps1]: Resolved

**Comment by rebelinux** [comment]
Everything looks good now. Thanks for the contribution!

**Review by rebelinux — APPROVED** [review]

**Pull request merged**